db-engine/src/main/java/com/splicemachine/db/impl/jdbc/EmbedPreparedStatement.java

```diff
@@ -1061,6 +1062,71 @@ public void addBatch() throws SQLException {
         }
     }
 
+    @Override
+    public int[] executeBatch() throws SQLException {
+        checkExecStatus();
+        synchronized (getConnectionSynchronization()) {
+            setupContextStack();
+            int i = 0;
```

Comment:
What is "I" here for? It doesn't seem to be very useful.

Reply:
Yep, bad copy paste.